### ISSUE TITLE
Ball model names are not translated

### ISSUE THREAD

**Issue by Wuzzy2** [Neverball/neverball#114]
In Neverball 1.6.0, the names of custom ball models are not translated (but should).

**Comment by ChallengeSY** [comment]
Intentional. Ball models are filenames, as are replay names and level set identifiers. Parasti states that it is simpler and less error-prone to keep filenames in ASCII, so filenames are intentionally non-translatable.

It is theoretically possible to refit in support, provided that "displayed names" independent of filenames are also added in.

**Comment by Wuzzy2** [comment]
Yeah, I support the idea of displaying the names indepently from file name. I do not propose to change any file name, just use them internally.

So for instance, “cheese-ball” may be _displayed_ as “Cheese Ball” in English, “Käseball” in German, etc.
